public/app/plugins/datasource/cloudwatch/components/ConfigEditor.tsx

```diff
@@ -0,0 +1,390 @@
+import React, { PureComponent, ChangeEvent } from 'react';
+import { FormLabel, Select, Input, Button } from '@grafana/ui';
+import { DataSourcePluginOptionsEditorProps, DataSourceSettings } from '@grafana/data';
+import { SelectableValue } from '@grafana/data';
+import { getDatasourceSrv } from 'app/features/plugins/datasource_srv';
+import CloudWatchDatasource from '../datasource';
+import { CloudWatchOptions } from '../types';
+
+export type Props = DataSourcePluginOptionsEditorProps<any>;
```

Comment:
Rather than <any> this should be your <CloudWatchOptions >... BUT needs a seperate object for the secure fields